node/src/Worker.ts

```diff
@@ -203,7 +202,6 @@ export type WorkerDump = {
 
 export type WorkerEvents = {
 	died: [Error];
-	subprocessclose: [];
```

Comment:
This is a documented event, why is it being removed? I mean, we still may need to know when the native addon and its worker thread are closed, am I wrong?

Reply:
They are never closed by themselves. The only way to close them is via `worker.close()` method in worker.ts, which sends a `WORKER_CLOSE` request to the worker. That should release all the libuv resources. Once that happens the thread running `mediasoup_worker_run()` will close, and when the JS `worker` instance is garbage collected, the native instance will be destroyed.

Reply:
I still have to do more testing on the closure part BTW.